docs/5.api/rpc/transactions.md

```diff
@@ -18,31 +18,11 @@ The RPC API enables you to send transactions and query their status.
 
 - method: `send_tx`
 - params: 
-  - SignedTransaction encoded in base64
-  - [Optional] `wait_until`: the required minimal execution level. It can be one of the listed below. The default value is `FINAL`.
+  - `signed_tx_base64`: SignedTransaction encoded in base64
+  - [Optional] `wait_until`: the required minimal execution level. Read more [here](/api/rpc/transactions#tx-status-result). The default value is `EXECUTED_OPTIMISTIC`.
```

Comment:
Please check if the links work, I have no idea how to check this locally, I just copy pasted the link format somewhere